content/articles/2017-08-08-contributing.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Contributing Code to the Tidyverse
+slug: contributing
+date: 2018-08-08
+categories: [Other]
+---
+
+Contributing code to open source projects can be intimidating. These projects
+are often widely used and have well known maintainers. Contributing code and
+having it accepted seems an almost insurmountable task.
+
+However if you follow a few simple strategies you can have your code accepted
+into even the most popular projects in the [tidyverse](https://tidyverse.org).
+
+### Don't Contribute Code at all ###
+
+The easiest way to contribute to an open source package is not to contribute
+code at all. Find a typo in the documentation, add a reproducible
+example to an open issue without one, post a solution to a question in an
+ issue, on [twitter](https://twitter.com/search?q=%23rstats) or
+[stackoverflow](https://stackoverflow.com/questions/tagged/r). These types of
+contributions are among the easiest things for maintainers to review and
+accept, so it is a great place to start getting used to the contribution
+process.
+
+### Read CONTRIBUTING.md ###
+
+Often projects will have a `CONTRIBUTING.md` document that has instructions for
+contributing to the project. These are guidelines the maintainers would like
+contributors to adhere to and exist to make the process flow more smoothly. As a
+contributor you should try to make accepting your code as easy as you
+can, this greatly increases the chance your contribution will be accepted.
```

Comment:
... These are not currently widespread in the tidyverse, but it's something we'll be working on in the future!